src/coreclr/inc/corinfo.h

```diff
@@ -3244,6 +3244,13 @@ class ICorDynamicInfo : public ICorStaticInfo
                     bool                    ignoreMovableObjects = true
                     ) = 0;
 
+    virtual bool getObjectData(
```

Comment:
Nit: Both methods return the same thing, but one calls it Value and the other calls Data. Can we use the same name for the thing that the method returns?

Some ideas:
- `readStaticField` + `readObject`
- `readStaticFieldData` + `readObjectData`
- `getStaticFieldContent` + `getObjectContent`

(`getReadonlyStaticFieldValue` method does not seem to be strictly limited to readonly static fields, so the name can be relaxed).

Reply:
Decided to go with:
```
getObjectType -> getObjectContent; 
getReadonlyStaticFieldValue -> getStaticFieldContent
```
Put a condition to `getObjectContent` to give up on types with GC pointers - want to have a real use case to test properly if you don't mind.